src/specs/models/WorkRequest.v2.yaml

```diff
@@ -4,15 +4,12 @@ description: This is the schema for the event emitted by the UI when certain wor
 properties:
   ETag:
     type: string
-    pattern: '^[a-fA-F0-9]+$'
+    pattern: "^[a-fA-F0-9]+$"
     description: A UUID generated by the client submitting the work. Will be used to map `WorkResponse` objects to the request submitted.
```

Comment:
this should be deleted from the spec, because the UI no longer sends to Etag to the api, right?

Reply:
this is not correct because that `WorkRequest.v2.yaml` is the one the API sends to the worker. And the Etag is there (generated by the API). See `src/api.v2/events/handleWorkRequest.js:35`

Reply:
makes sense